int8 qr_nums
string type
string data
geometry_msgs/Point[] qr_points
